File: ssaExample/ast_File.java
import java.util.Map;

public class ast_File {

	public ast_CommentGroup Doc;
	public Integer Package;
	public ast_Ident Name;
	public Object[] Decls;
	public Integer FileStart;
	public Integer FileEnd;
	public ast_Scope Scope;
	public ast_ImportSpec[] Imports;
	public ast_Ident[] Unresolved;
	public ast_CommentGroup[] Comments;
	public String GoVersion;
}

File: ssaExample/ast_File.kt
class ast_File {

	var Doc: ast_CommentGroup? = null
	var Package: Int? = null
	var Name: ast_Ident? = null
	var Decls: Array<Any>? = null
	var FileStart: Int? = null
	var FileEnd: Int? = null
	var Scope: ast_Scope? = null
	var Imports: Array<ast_ImportSpec>? = null
	var Unresolved: Array<ast_Ident>? = null
	var Comments: Array<ast_CommentGroup>? = null
	var GoVersion: String? = null
}
